ui/config.ts

```diff
@@ -0,0 +1 @@
+export const API_URL='INSERT_API_HERE'
```

Comment:
You should pick this up from `process.env.API_URL`, or something along those lines. Need some way for a deploy to not fail.

Reply:
I can't. There's no option. It isn't passed properly. I tried everything.